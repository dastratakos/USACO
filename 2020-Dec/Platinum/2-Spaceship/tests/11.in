60 60 60
111111111011110101111011110001011111001011111010111111111011
101101001110110101111110110101111010011111000001110111100110
010110001010111101111000001111100100011100011111111101011100
010101111100101110000000111001111011111101100110101111111101
111011111111111010101001101111101111111110011111110011001100
110111101100111100111110111011111010101111111111110100110011
001011001111111111111111111111001100100011001110111101100011
111100101111110111000011100111001111111110011011111111100001
011100010111111111111101000001110101111111101110010110101010
110111101111011010011011010001101000011011101011111011111110
111111111011101101111011111100101111110110101111111111110100
110101111110001111101111110101101101010111111111110111110111
001001101111011010101111110011010110111100101111000100101101
011111101001001011110011100011011110101110011010111101110111
111001101110010111111101010111111110110101110101110100110001
001011101101101111111111111110111011101111101101001111100111
001001011011010110111011100001001101011011001011011110011111
111111101011111111101110000101010101000100101010011111111010
101101111111111011010011111101111111111110110111101101111010
101001101111011101011011101001111101001000111011011100111111
111011111011001111111110010011111101111111011110010011001011
100011111111011111111101100111101011101111011101011111111001
001001010110110110101001111101111010111100101110101111111101
110111110011100011000010111101101110101101000011111101101011
010110101101011010011010111101010101111111011010001100110011
111011110110100010110010011111101110101110101111101101110111
101011101110111011111111111101000011011101111110010011011111
110110111100111101011101011101110011010001111011001111110101
111011110000111111111111001110111110111011100101010111101101
110111001111110111011110110111101011100111111111010011011010
110100101111010101111010011101010011111100010101111101111100
111011110110101110010111101011110011110110011110101111110111
111001010011111110110110100011001110111101111110101001010010
001111111101101011010111110111110011011110110111000111110110
010000011110000111101111111111101000111111101011101110111110
011101010110000111101110011001110110110110111111110111110001
101111111001110001110111101101111111001111111110100110110101
111111000010001111111110111001001101010111101111111100010111
100111001111011011110101010010111101101011011010111101011101
011111110110011001010000010110101000111101110110011110110010
010111101011011011111111101001110111100111111101000000110110
111111111111010111010111101001111011110111101001111101101110
110101101110000011000110010010100001110100011001011111101101
100100111000111101101011010110101111010110011110101100111011
111110010111111101111111001100101110111011111111001111010010
010110001101111111100111111110010111100011111111010100000111
010100101111101111111001011111010001110011111111101111101100
110011111100111110100111010111001000001011111110111111111111
110111100110011111011000010110111101111101110001000111111111
011010011011010111111010111011110101111111111100110100111001
110101011111011010111100101111111101011110100110110111111111
111100110111001111101011111011110100111111110111011001111111
101011001011111100111011100111111010111111000100100010011011
001011011110011100111011111111001011001110001111011001001101
100101101101111111011011110101101011010111101001011001011011
110011011101111111111110110111011110111111111111000011011111
000111111110100111011111100010001100101111011011111110010101
111111000111001101011110111000011111110111110111110101101111
101111011001111111011111111011111011011001010011011010111001
111011001110010010101010101011110101101011111111010001101111
59 3 60 33
59 41 60 9
59 41 60 52
59 2 60 1
59 27 60 6
59 55 60 16
59 58 60 39
59 47 60 20
59 26 60 46
59 25 60 27
59 4 60 26
59 6 60 39
59 35 60 43
59 55 60 36
59 12 60 58
59 12 60 47
59 56 60 11
59 36 60 21
59 60 60 38
59 47 60 60
59 59 60 42
59 39 60 44
59 32 60 51
59 50 60 20
59 8 60 26
59 53 60 15
59 39 60 17
59 50 60 11
59 2 60 24
59 46 60 27
59 23 60 54
59 48 60 5
59 16 60 3
59 9 60 42
59 17 60 16
59 59 60 16
59 20 60 21
59 6 60 48
59 41 60 38
59 44 60 13
59 47 60 41
59 17 60 26
59 23 60 24
59 57 60 17
59 2 60 21
59 9 60 20
59 16 60 35
59 52 60 54
59 18 60 43
59 27 60 53
59 44 60 17
59 47 60 45
59 5 60 4
59 31 60 46
59 25 60 14
59 44 60 9
59 18 60 24
59 34 60 23
59 35 60 4
59 25 60 12
