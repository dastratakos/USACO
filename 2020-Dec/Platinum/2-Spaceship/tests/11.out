705729571
482306065
834387064
916925094
765359395
242043081
557939924
551188446
446577089
525183156
179286212
88899407
485073392
413801754
703539712
455683368
555335243
201630394
757031532
197904505
909278462
979693168
672510437
471471002
854057178
458752231
568542639
839683999
134242526
514389118
777541428
471088836
388379612
741695853
379663871
152451241
631927453
105268426
651278886
602062735
189300166
706930069
4586700
401572056
585795630
876036020
297599537
563067772
706157006
867170842
974782508
579049940
449483665
371112199
31064462
427751571
184259428
531911670
326005897
913736763
